parachains/runtimes/assets/westmint/src/weights/pallet_uniques.rs

```diff
@@ -74,13 +74,14 @@ impl<T: frame_system::Config> pallet_uniques::WeightInfo for WeightInfo<T> {
 	/// The range of component `m` is `[0, 1000]`.
 	/// The range of component `a` is `[0, 1000]`.
 	fn destroy(n: u32, m: u32, a: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 13_000
-			.saturating_add(Weight::from_ref_time(10_002_000 as u64).saturating_mul(n as u64))
-			// Standard Error: 13_000
-			.saturating_add(Weight::from_ref_time(1_796_000 as u64).saturating_mul(m as u64))
-			// Standard Error: 13_000
-			.saturating_add(Weight::from_ref_time(1_646_000 as u64).saturating_mul(a as u64))
+		// Minimum execution time: 2_518_363 nanoseconds.
+		Weight::from_ref_time(2_538_933_000 as u64)
```

Comment:
big jump from 0

Reply:
Yeah, just checked the changes in last release, this one was applied on Statemine/t but not on Westmint